src/features/events/filters/eventDropdownFilter.js

```diff
@@ -96,68 +96,53 @@ const Badge = styled.span`
   `};
 `
 
-class EventDropdownFilter extends Component {
-  state = {
-    isOpen: false,
-  }
-
-  shouldComponentUpdate(nextProps, nextState) {
-    return this.state.isOpen !== nextState.isOpen
-  }
-
-  handleClickOutside = () => {
-    if (this.props.filterName === this.props.filterOpen) {
-      this.setState({ isOpen: false }, () => {
-        this.props.closeSiblingFilters(this.props.filterName, this.state.isOpen)
-      })
-    }
-  }
+const EventDropdownFilter = React.memo(
```

Comment:
I saw your comment about this in the description for the PR. This would not be a swapout for `shouldComponentUpdate`. What you need to identify is what role the previous function was serving, and if it's a) actually required now, and b) it functions in exactly the same way as before with whatever you end up implementing (if anything). A good starting point would be to get some test coverage (see other comment).